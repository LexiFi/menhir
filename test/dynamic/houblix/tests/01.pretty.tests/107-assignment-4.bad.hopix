let main =
  \ -> x := 0