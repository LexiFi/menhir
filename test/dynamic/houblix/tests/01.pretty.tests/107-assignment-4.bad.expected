Line 2, characters 4-6: Error (parsing)
  Syntax error.
